src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -40,7 +44,26 @@
      * София - 900</p>
      */
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo)
-            throws Exception {
-        return null;
+            throws IllegalDateParametersException, ParseException {
+        SimpleDateFormat dateFormat = new SimpleDateFormat("dd.MM.yyyy");
```

Comment:
formatter should be a constant field as it's better for code scaling. BTW, it was in the checklist, didn't you see that?

Reply:
Ok, I've seen but forgot it